src/DependencyInjection/DI/src/ServiceProviderOptions.cs

```diff
@@ -18,6 +18,11 @@ public class ServiceProviderOptions
         /// </summary>
         public bool ValidateScopes { get; set; }
 
+        /// <summary>
+        /// <c>true</c> to perform check verifying that all services can be created during <code>BuildServiceProvider</code> call; otherwise <c>false</c>.
```

Comment:
Might be worth mentioning that ValidateOnBuild defaults to false. We could do the same to ValidateScopes for consistency.

Reply:
Maybe this should also mention that this doesn't verify open generic services.